krun.py

```diff
@@ -203,13 +200,41 @@ def get_overall_time_estimate_formatter(self):
     def add_eta_info(self, key, exec_time):
         self.eta_estimates[key].append(exec_time)
 
+    def build_schedule(self, config, current):
```

Comment:
I would love one, but I couldn't think of anything sensible. Maybe `previous_schedule`, `completed_jobs`?